onnxruntime/core/providers/webnn/builders/helper.cc

```diff
@@ -99,6 +99,84 @@ bool IsTensorShapeSupported(const NodeArg& node_arg, const std::string& parent_n
   return true;
 }
 
+/**
+ * Checks if all input tensor ranks (dimension counts) of the given node are supported by WebNN specifications.
```

Comment:
Thanks for such detailed comments for this function, to align with other functions, I think we can just simplified it with a single sentence:

```
Check if all input tensor ranks of the given node are supported by WebNN.
```